sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2709,31 +2709,32 @@ class HiveDDLSuite
     assert(compression === actualCompression)
   }
 
-  Seq(("orc", "ZLIB"), ("parquet", "GZIP")).foreach { case (fileFormat, compression) =>
-    test(s"SPARK-22158 convertMetastore should not ignore table property - $fileFormat") {
-      withSQLConf(CONVERT_METASTORE_ORC.key -> "true", CONVERT_METASTORE_PARQUET.key -> "true") {
-        withTable("t") {
-          withTempPath { path =>
-            sql(
-              s"""
-                |CREATE TABLE t(id int) USING hive
-                |OPTIONS(fileFormat '$fileFormat', compression '$compression')
-                |LOCATION '${path.toURI}'
+  Seq(("orc", "ZLIB"), ("parquet", ParquetCompressionCodec.GZIP.name)).foreach {
```

Comment:
nit: Make `Seq(("orc", "ZLIB"), ("parquet", ParquetCompressionCodec.GZIP.name))` a variable, and then use `seq..foreach { case (fileFormat, compression) =>`. Would the code below need to be reformatted?

Reply:
For reduce the change here, let's use
```
Seq(
  ("orc", "ZLIB"),
  ("parquet", ParquetCompressionCodec.GZIP.name)).foreach { case (fileFormat, compression) =>
```